src/store/useUserStore.ts

```
@@ -14,6 +16,12 @@ export const useUserStore = create<UserStateTypes>((set) => ({
     typeof window !== 'undefined' ? localStorage.getItem('accessToken') : null,
   userName:
     typeof window !== 'undefined' ? localStorage.getItem('userName') : null,
+  userId:
+    typeof window !== 'undefined'
```

ESLint

[error] Replace `'undefined'` with `"undefined"`

(prettier/prettier)